example_scripts/example_runScripts.py

```diff
@@ -77,7 +78,7 @@
 # problem_sets = ["Neutron_data", "NIST/average_difficulty"]
 # problem_sets = ["CUTEst", "Muon_data", "Neutron_data", "NIST/average_difficulty", "NIST/high_difficulty", "NIST/low_difficulty"]
 
-problem_sets = ["NIST/low_difficulty"]
+problem_sets = ["Muon_data"]
```

Comment:
@AtomAnu did you mean to change problem set up Muon_data in this script? I.e. with problem_sets = Muon_data this script with not run without mantid